src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/DocumentTag.java

```diff
@@ -75,6 +75,7 @@ public enum DocumentTag {
     INTERNAL_NON_STANDARD_DIRECTION_RESPONDENT_LETTER("internalNonStandardDirectionToRespondentLetter", CaseType.ASYLUM),
     INTERNAL_HO_CHANGE_DIRECTION_DUE_DATE_LETTER("internalHoChangeDirectionDueDateLetter", CaseType.ASYLUM),
     INTERNAL_REINSTATE_APPEAL_LETTER("internalReinstateAppealLetter", CaseType.ASYLUM),
+    INTERNAL_ADJOURN_HEARING_WITHOUT_DATE_LETTER("internalAdjournHearingWithoutDateLetter", CaseType.ASYLUM),
```

Comment:
https://github.com/hmcts/ia-case-notifications-api/pull/1054/files#diff-9fda946ee08cb3b88ebda6b166a8c20f62cd5933d70b87c890d93492547472a5  created tag in notifications